test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/RequestTests.cs

```diff
@@ -76,6 +80,72 @@ public class RequestTests
             }
         }
 
+        [Fact]
+        public async Task LargeMultipartUpload()
```

Comment:
Yeah, I meant something along the lines of [`[OSSkipCondition]`](https://github.com/aspnet/Testing/blob/dev/src/Microsoft.AspNetCore.Testing/xunit/OSSkipConditionAttribute.cs)